### ISSUE TITLE
build for pinephone

### ISSUE THREAD

**Issue by gemlog** [pdf/kdeconnect-chrome-extension#45]
I don't really expect it for such a tiny community, but I thought I'd ask anyhow :-) 
Also, since I've been using it for so long on android, I was very surprised with ff only listed about 1k users? I would have thought it was vastly more popular than that.
Thanks! and thanks for all the time I've had use of it on android!

**Comment by pdf** [comment]
Just to be sure, you're looking for the browser extension, and not the full KDE Connect? Because I'm surprised to hear that you had this extension working on Android.

**Comment by gemlog** [comment]
You are right! I've been using ff klar for so long with kdeconnect I thought I must and had forgotten. KDEconnect works fine on the pinephone now, but not in FF, so I thought I needed to add an extension and assumed it was yours. I wonder why sharing to kde works in android klar and not ff on pinephone. Sorry I bothered you. I do have your extension installed on my amd64 pc. I'll figure something out - eventually ;-)
paul

**Comment by pdf** [comment]
I suspect the Android builds have extra bits that enable the native Android sharing mechanism, but no such thing exists on pinephone, since that's basically just Linux with a phone-sized window-manager/desktop-environment (AIUI). So yeah, a browser extension like one would maybe help (though I don't know what the UI for interacting with extensions is like on that form-factor). It might just work if built for aarch64 (and you can run stuff from a terminal), but beyond that I'm not sure how much help I'm likely to be able to provide in the short term.

**Comment by gemlog** [comment]
So I thought it would just be recompiling for arm vs amd64. PC your extension works perfectly. If you can compile it for arm, it should work no? Mind you, I have no idea what a massive headache installing a toolchain for arm might be. As I said, I'll muddle through without it (e.g. cp/pasta a link to email myself). It's just a nice added convenience having your extension in the browser.

**Comment by pdf** [comment]
Another kind individual is helping me with Windows support, when that goes live I'll be changing the way I do releases, and I should be able to include an aarch64 build when that happens, we'll see if that works without additional modifications.

**Comment by gemlog** [comment]
Cool! I look forward to testing it, if you remember me then.

**Comment by gemlog** [comment]
thank you for your time helping me. And thank you for all the good use I've already had from your extension on amd64. I don't have much money and I use a LOT of free oss software, so I am grateful. I just don't get around to every project... :-( 
I didn't see a link for you, so I bought you part of an operation - I hope that's ok.
https://files.kalum.ca/f.php?h=3Ub_d-qZ&p=1
https://watsi.org/profile/0386e584b223-bekalu

**Comment by pdf** [comment]
Thank you, that's very kind, and much appreciated.
